.travis.yml

```diff
@@ -102,4 +102,5 @@ deploy:
   on:
     repo: microsoft/DirectXShaderCompiler
     branch: master
+    condition: $CC=clang
```

Comment:
This didn't work.  I don't know anything about the syntax used here, but should it be `==`?

Reply:
The Travis [documentation](https://docs.travis-ci.com/user/deployment) says we should use `=`. 
It looks like Travis is **not** uploading the artifacts (for any configuration) to the cloud storage (the bucket is still empty). And the Travis logs show no errors or warning. So there's no way to figure out what's going wrong... I'll abandon this effort for now to stop the linux builds from failing.